.github/workflows/client.yml

```diff
@@ -38,6 +38,14 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
+      - name: Load environment variables
+        uses: keep-network/load-env-variables@main # TODO: use @v1 once availible
+        env: 
+          CI_GITHUB_TOKEN: ${{ secrets.CI_GITHUB_TOKEN }}
+        with:
+          environment: 'ropsten'
+          ref: 'master'
```

Comment:
Could `master` be a default value for `ref` defined in the action? We wouldn't have to provide anything here. Non-default `ref` would be super rare and used mostly to test some new configurations.
```suggestion
```